deepchem/data/datasets.py

```diff
@@ -444,13 +452,13 @@ def get_statistics(self, X_stats=True, y_stats=True):
     elif X_stats and y_stats:
       return X_means, X_stds, y_means, y_stds
     else:
-      return None
+      return tuple()
 
   def make_tf_dataset(self,
-                      batch_size=100,
-                      epochs=1,
-                      deterministic=False,
-                      pad_batches=False):
+                      batch_size: int = 100,
+                      epochs: int = 1,
+                      deterministic: bool = False,
+                      pad_batches: bool = False):
```

Comment:
Should we add a return type here?

Reply:
Same here.  Declaring the return type would require importing TensorFlow at the top level.